diopter/utils.py

```diff
@@ -55,14 +57,25 @@ def run_cmd_async(
 
     if isinstance(cmd, list):
         cmd = " ".join(cmd)
-    return subprocess.Popen(
-        shlex.split(cmd.replace('"', '\\"')),
-        cwd=str(working_dir),
-        env=env,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        **kwargs,
-    )
+
+    if stdout is not None and stderr is not None:
+        return subprocess.Popen(
+            shlex.split(cmd.replace('"', '\\"')),
+            cwd=str(working_dir),
+            env=env,
+            stdout=stdout,
+            stderr=stderr,
+            **kwargs,
+        )
+    else:
```

Comment:
Instead of an if else could you not do:
```if stdout is None:
      stdout = subprocess.PIPE
```
and the same for `stderr` (you may have to adjust the type annotation or introduce another local variable)

Reply:
Yeah that's the right way to do that, I just didn't want to add an extra import to everything...

Reply:
Another option is to do this inside the function: 

```
out = subprocess.PIPE if stdout is None else stdout
```
Then you don't need to adjust the signature and incur imports